app-media-image-capture.html

```diff
@@ -191,118 +202,144 @@
           },
 
           /**
-           * @type {Polymer.AppMedia.MeteringMode}
-           * @see https://www.w3.org/TR/image-capture/#PhotoSettings
+           * @type {Polymer.AppMedia.FillLightMode}
+           * @see https://w3c.github.io/mediacapture-image/##photosettings-section
            */
+          fillLightMode: {
+            type: String,
+            value: null
```

Comment:
do you need these default values to be `null` and not `undefined`? Do they trigger multi prop observers? (here and errwhere)